newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionOptionsEditor.js

```diff
@@ -202,54 +202,67 @@ export const ExtensionOptionsEditor = ({
               eventsFunctionsExtension.setCategory(category);
               forceUpdate();
             }}
+            // TODO Sort by translated value.
```

Comment:
Are those todos meant for this PR or a later one?

Reply:
Not sure it's that important to be honest.
But if we want to do it, it'd have to be done inside the autocomplete, after computing all the translations and re-ordering the dataSource based on the results.